omero/institution/getting-started.html

```diff
@@ -9,8 +9,108 @@
+                    <li><a href="https://docs.openmicroscopy.org/omero/latest/sysadmins/index.html" target="_blank">Server upgrades</a></li>
+                    <li><a href="https://docs.openmicroscopy.org/omero/latest/sysadmins/index.html" target="_blank">Server configureation optimization or changes</a></li>
+                    <li>Installation of <a href="{{ site.baseurl }}/omero/apps/" target="_blank">OMERO Apps and extensions</a></li>
+                    <li><a href="https://docs.openmicroscopy.org/omero/latest/sysadmins/server-security.html" target="_blank">Server security and firewalls</a></li>
+                </ul>
+                <h5>OMERO manager person(s)</h5>
+                <ul class="dev-ops-links">
+                    This person is a go-between the users of OMERO and the system administrator of the server. Very often this might be a microscopy facility manager.
+                    Typical tasks:
+                    <li><a href="https://omero-guides.readthedocs.io/en/latest/example_facility_manager.html" target="_blank">Basic walkthrough</a></li>
+                    <li><a href="https://omero-guides.readthedocs.io/en/latest/introduction/docs/group-user-management.html" target="_blank">Management of Groups/Users in OMERO</a></li>
+                    <li>Collating feedback from users</li>
+                    <li><a href="https://omero-guides.readthedocs.io/en/latest/example_facility_manager.html#get-users-started" target="_blank">Introducing new users to OMERO</a></li>
+                    <li>Liaise with system administrator for features which require server configuration changes or installation of <a href="{{ site.baseurl }}/omero/apps/" target="_blank">OMERO Apps</a> and extensions.</li>
+                    <li>According to local needs, design and upload of new OMERO.scripts, possibly in cooperation with image analyst.</li>
+                    <li>Design and with system administrator implement an <a href="https://docs.openmicroscopy.org/omero/5.6.3/sysadmins/import-scenarios.html" target="_blank">import pipeline </a>, selecting the best possible <a href="{{ site.baseurl }}/omero/features/import/" target="_blank"> import setup</a> for the institution.</li>
+                </ul>
+            </div>
+        </div>
+        <!-- end What is needed -->
+        <a id="contacts"></a>
+        <hr class="whitespace">
+        <div class="row column text-center">
+            <h3>Key OMERO contacts</h3>
+        </div>
+        <hr>
+        <div class="row">
+                <h5>Contact OME Team</h5>
+                <ul class="dev-ops-links">
+                    <li><a href="{{ site.baseurl }}/support/" target="_blank">Main contact to OME Team is via image.sc</a></li>
+                    <li><a href="{{ site.baseurl }}/support/" target="_blank">Check other means of support on this webpage</a></li>
```

Comment:
These both link to the same place. Maybe the first should link to image.sc

Reply:
fixed in https://github.com/ome/www.openmicroscopy.org/pull/462/commits/b34402411fec182b880b40934783bd667f93945f